src/db/models.py

```diff
@@ -1,24 +1,64 @@
 from . import connection
+import psycopg2.sql as sql
 
 class ApplicationsModel:
+    TableName = "applications"
+    Model = {}
+
+    @classmethod
+    def setModel(cls, fieldnames, sample_data):
+        for field in fieldnames:
+            cls.Model[field] = str # force everything into string, deal with on client
+        
+        cls.Model["score"] = int
+        
+        conn = connection.get_connection()
+        cursor = conn.cursor()
+        try:
+            cursor.execute('DROP TABLE IF EXISTS %s' % ApplicationsModel.TableName)
```

Comment:
We might want to extract these two calls into separate functions, but fine for now

Reply:
in the other PR